tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-training-apis.yml

```diff
@@ -0,0 +1,48 @@
+trigger: none
```

Comment:
Was the addition of this yml intentional or was it meant to be a replacement for tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-on-device-training.yml ?

Reply:
I intentionally added a duplicate file. Once this PR is checked in I will update the devops pipeline to use this yml and then delete the original one... If there is a better way to handle this let me know :)

Reply:
there is a way to do it which requires manually running the pipeline for your PR (by updating the yml file being used for the build). As soon as your PR pipeline is triggered, you can change the pipeline back to using the one in main.
Once the PR is merged, you could change the yml file again.

But up to you, whatever is more convenient.